source/whats-new.txt

```diff
@@ -171,6 +172,37 @@ This release includes the following improvements and fixes:
 
 - The ``Collection.Clone()`` method does not return an error.
 
+.. _golang-version-1.17:
+
+What's New in 1.17
+------------------
+
+.. important::
+
+   v1.17 is the final planned v1.x version release. This version will
+   receive bug fixes, but future development and features will be
```

Comment:
We will also fix security issues such as CVEs which are different from bugs. Suggest updating the language here: 

> receive security and bug fixes, but future development and features will be